crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -191,12 +190,11 @@ impl CallContractCommand {
 		cli.warning("NOTE: contract has not yet been built.")?;
 		let spinner = spinner();
 		spinner.start("Building contract in RELEASE mode...");
-		let result = match build_smart_contract(&project_path, true, Verbosity::Quiet) {
+		let result = match build_smart_contract(&project_path, true, Verbosity::Quiet, None) {
```

Comment:
Is it correct not to pass the metadata spec here?

Reply:
`cargo-contract` doesn’t support deploying or calling contracts that use Solidity metadata yet.
In this specific case, where is building the contract only before pop call, I think it makes sense to use the default (ink!) metadata for the build process.

Reply:
Perfect then 👌